dashboard/app/models/pd/workshop.rb

```diff
@@ -347,6 +351,16 @@ def year
     sessions.order(:start).first.start.strftime('%Y')
   end
 
+  # Suppress 3 and 10-day reminders for certain workshops
+  def suppress_reminders?
```

Comment:
Not strongly, I just figured it was worth looking at from both sides. Your reasoning has convinced me!